tb/dpi/remote_bitbang.cc

```diff
@@ -43,7 +43,7 @@ remote_bitbang_t::remote_bitbang_t(uint16_t port) :
   memset(&addr, 0, sizeof(addr));
   addr.sin_family = AF_INET;
   addr.sin_addr.s_addr = INADDR_ANY;
-  addr.sin_port = htons(port);
+  addr.sin_port = htons(2222);
```

Comment:
We should keep this parameterizable...

Reply:
By default the port is random, this is probably not intentional, no doubt there is a simpler fix.